src/methods/aggregate.jl

```diff
@@ -513,4 +515,4 @@ function _view_disaggregate(A, scale)
     dims_ = disaggregate.((Center(),), dims(A), intscale)
     indices = map((a, i) -> repeat(a; inner=i), axes(A), intscale)
     return rebuild(A; data=view(parent(A), indices...), dims=dims_)
-end
\ No newline at end of file
+end
```

Comment:
this was completely github's fault, this PR was online only :D

Reply:
Microsoft adding their awful line endings